### PR TITLE
Auto Query Fixer: develop the java client for ZetaSQL Helper

### PR DESCRIPTION
**Pull request by mingen-pan**
Develop the java client to call RPC functions from the ZetaSQL Helper server.

A dokcer controller is also added to let the client automatically start and connect to the server

### CODE REVIEW

File: tools/zetasql_helper/java/com/google/bigquery/utils/zetasqlhelper/Client.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+import io.grpc.Channel;
+
+/**
+ * Client to establish connections with ZetaSQL Helper server.
+ */
+public class Client {
+
+    private static ZetaSqlHelperLocalServiceGrpc.ZetaSqlHelperLocalServiceBlockingStub stub;
+    private static int port = 50051;
```
Comment (kikkyo) [inline tools/zetasql_helper/java/com/google/bigquery/utils/zetasqlhelper/Client.java:11]: Can we use random port here?

File: tools/zetasql_helper/java/com/google/bigquery/utils/zetasqlhelper/LocalServiceDockerProvider.java [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+ * if a server container is started, start a server and connect to a server.
+ */
+public class LocalServiceDockerProvider implements LocalServiceProvider {
+
+    private final static Integer SERVER_PORT = 50051;
+    private final static String LOCAL_HOST = "localhost:";
+    private static String IMAGE = "gcr.io/sql-gravity-internship/zetasql-helper:1.0";
+
+    private final static String RUNNING = "running";
+    // waiting time for starting server. If the server container con not start in this period,
```
Comment (kikkyo) [inline tools/zetasql_helper/java/com/google/bigquery/utils/zetasqlhelper/LocalServiceDockerProvider.java:28]: nit: "can" not

File: tools/zetasql_helper/java/com/google/bigquery/utils/zetasqlhelper/LocalServiceDockerProvider.java [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+    /**
+     * Connect to the server at the given port number.
+     *
+     * @param port port number
+     * @return Channel to the server
+     */
+    @Override
+    public Channel connect(final Integer port) {
+        try {
+            // if the server is not active, try to start it.
```
Comment (kikkyo) [inline tools/zetasql_helper/java/com/google/bigquery/utils/zetasqlhelper/LocalServiceDockerProvider.java:100]: nit: If

**Review by kikkyo — APPROVED** [review]

File: tools/zetasql_helper/java/com/google/bigquery/utils/zetasqlhelper/Client.java (hunk 1 above)
Comment (mingen-pan) [reply tools/zetasql_helper/java/com/google/bigquery/utils/zetasqlhelper/Client.java:11]: The default one is 50051, but users can always set their own port using setPort method

**Pull request merged**